lib/ui/painting.dart

```diff
@@ -1595,6 +1637,125 @@ class Image extends NativeFieldWrapperClass2 {
   ///
   /// Returns a future that completes with the binary image data or an error
   /// if encoding fails.
+  Future<ByteData?> toByteData({ImageByteFormat format = ImageByteFormat.rawRgba}) {
+    assert(!_disposed && !_image._disposed);
+    return _image.toByteData(format: format);
+  }
+
+  /// If asserts are enabled, returns the [StackTrace]s of each open handle from
+  /// [clone], in creation order.
+  ///
+  /// If asserts are disabled, this method always returns null.
+  List<StackTrace>? debugGetOpenHandleStackTraces() {
+    List<StackTrace>? stacks;
```

Comment:
It looks like there's an opportunity with this new API to avoid allowing nulls. How about initializing this to `List.empty()` and getting rid of the `?` from the return type?

Reply:
I think we usually make the `debug*` methods return null in the framework, but I'm not opposed to this.

Is there an advantage to this? This method will never return a meaningful value in release.

Reply:
If there's already a convention around this, then sticking with that for this PR sgtm.

The advantage is just avoiding mistakes around `null` and having the `?`'s propagate around everywhere.